### ISSUE TITLE
Feature Request : Add switch to export as HTML content

### ISSUE THREAD

**Issue by SamLefebvre** [twibiral/obsidian-execute-code#13]
Hi, 
Is it possible to add a switch near the button Run or in the menu to be able to export as html ?

I have found that if I change this line https://github.com/twibiral/obsidian-execute-code/blob/fcfc7baf8b34935eb17e2327b0c600ed813ef40c/Outputter.ts#L47
To 
```javascript
this.stdoutElem.innerHTML = (this.stdoutText);
```
It can export as a html content and not be interpreted only as text.

**Why would I like this new feature ?**

To be able to export images. With that, it will be possible to execute code and have the output (like matplotlib graphs) directly in Obsidian (I tried and it work).

Here a screenshot of the content as html.
![image](https://user-images.githubusercontent.com/8534618/169705989-cc278f7a-31e1-4e75-893c-e5114e71817b.png)

Thank you.

**Comment by twibiral** [comment]
That's a great idea @SamLefebvre !
I will work in it but feel free to implement your own solution and make a merge request.
What do you think is the best way to show matplotlib plots?

**Comment by SamLefebvre** [comment]
With matplotlib we can convert the result as base64 and show it as html, this is how I did it: 

The code is from **Rex D** https://stackoverflow.com/a/63381737 with minor modifications.
Here the result: 

![image](https://user-images.githubusercontent.com/8534618/169707405-fd39f9b4-26fd-44ed-a229-0a7e6fe839b6.png)

and the code:

``` python
#plt.show() 

import io
import base64
s = io.BytesIO()
plt.plot()
plt.savefig(s, format='png', bbox_inches="tight")
plt.close()
s = base64.b64encode(s.getvalue()).decode("utf-8").replace("\n", "")
html =  '<img align="left" src="data:image/png;base64,%s">' % s

print (html)
```

**Comment by twibiral** [comment]
I love this!
It would also be great to inject your code automatically every time plt.show() is called. Either by exchanging the show() call directly with your code or by overwriting the method.

**Comment by twibiral** [comment]
Implemented in new version 0.8.0 in #26
